src/test/java/com/liferay/upgrades/analyzer/exporter/GamePlanProjectDependencyExporterTest.java

```diff
@@ -0,0 +1,57 @@
+package com.liferay.upgrades.analyzer.exporter;
+
+import com.liferay.upgrades.analyzer.project.dependency.exporter.GamePlanProjectDependencyExporter;
+import com.liferay.upgrades.analyzer.project.dependency.graph.builder.ProjectsDependencyGraph;
+import com.liferay.upgrades.analyzer.project.dependency.graph.builder.ProjectsDependencyGraphBuilder;
+import com.liferay.upgrades.analyzer.project.dependency.model.Project;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class GamePlanProjectDependencyExporterTest {
+
+    @Test
+    public void testExportGamePlan() throws IOException {
+        final var expectedMessage = "Game Plan generated at ";
+
+        ProjectsDependencyGraphBuilder projectsDependencyGraphBuilder = new ProjectsDependencyGraphBuilder();
+
+        ProjectsDependencyGraph projectsDependencyGraph = projectsDependencyGraphBuilder
+                .addProject(
+                        new Project("a"),
+                        Set.of())
+                .addProject(
+                        new Project("b"),
+                        Set.of())
+                .addProject(
+                        new Project("c"),
+                        Set.of()).build();
+
+        final var result = new GamePlanProjectDependencyExporter().export(projectsDependencyGraph);
+
+        Pattern pattern = Pattern.compile("Game Plan generated at (.+\\/(projects-\\d+\\.txt))");
+        Matcher matcher = pattern.matcher(result);
+
+        Assertions.assertTrue(result.contains("Game Plan generated at "), expectedMessage);
+
+        if (matcher.find()){
+            String fullPathString = matcher.group(1);
+
+            Path generatedGamePlanFilePath = Paths.get(fullPathString);
+
+            List<String> lines = Files.readAllLines(generatedGamePlanFilePath);
+
+            Assertions.assertNotNull(lines, "The lines read must not be null.");
```

Comment:
The test scope seems fine to me, but I think that you need to compare the content in the generated file. Just assert if the line is not null, seems to be insufficient to guarantee the correct behavior of the exporter. What do you think @linolaoj?

Reply:
The test generates a file with the following lines: 
```This project contains 3 projects with 1 level(s) of project dependencies.
Level 1 count: 3
	a 0
	b 0
	c 0
```

That's why, in addition to the non-null test, I added the comparison with one of the lines in the file:
```Assertions.assertTrue(lines.contains("Level 1 count: 3"));```

Reply:
I see your point now!! It's fine!!

Reply:
We can talk about it personally